sdk/appconfiguration/ci.yml

```diff
@@ -0,0 +1,25 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger:
+  branches:
+    include:
+      - master
+      - feature/*
+  paths:
+    include:
+      - sdk/appconfiguration/
+
+pr:
+  branches:
+    include:
+      - master
+      - feature/*
+  paths:
+    include:
+      - sdk/appconfiguration/
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+    parameters:
+      ServiceDirectory: app-configuration
```

Comment:
in this case the service directory should be `appconfiguration` ... it's the service part of the path `/sdk/appconfiguration/...`

Reply:
Thanks!  Will fix that